app/models/content_item.rb

```diff
@@ -37,6 +37,8 @@ def merge(to_merge)
   end
 
   def phase
-    @content_item_data["phase"]
+    @content_item_data.fetch("phase")
+  rescue KeyError => e
+    puts "Error searching content item: #{e}"
```

Comment:
Should we raise an error here instead of writing a message? How will that affect the way it's logged?